src/features/Home.js

```diff
@@ -67,13 +70,9 @@ export default class Home extends Component {
         OAuthToken: OAuthToken,
         calID: calID
       });
-      await Promise.all([
-        deleteUserFromDB(localStorage.getItem(appTokenKey)),
-        deleteUser()
-      ]);
-      localStorage.removeItem(appTokenKey);
-      this.props.history.push("/login");
-      console.log("user deleted from firebase");
+      await deleteUserFromDB(localStorage.getItem(appTokenKey));
```

Comment:
Why is there `deleteUserFromDB` and `deleteUser`?

Reply:
deleteUser removes the user from Firebase Auth. deleteUserFromDB deletes the user from Firestore.

When a user first signs up, they are initialized in Firebase Auth and then those details (email, Firebase Auth generated Unique ID) are saved in Firebase Firestore (users collection). A calendar is created after the user is created and then the user's calendar id is saved under their user id in Firestore.

Reply:
*I'll rename deleteUser to deleteUserFromAuth to make it more clear